client/ayon_core/tools/push_to_project/ui/window.py

```diff
@@ -424,6 +436,10 @@ def _on_original_names_change(self, state: int) -> None:
         use_original_name = bool(state)
         self._invalidate_use_original_names(use_original_name)
 
+    def _on_version_up_checkbox_change(self, state: int) -> None:
+        self._controller._version_up = bool(state)
```

Comment:
Please add `set_version_up` to controller and call it. Never touch private attributes or methods.